src/hooks/useToggle.ts

```diff
@@ -0,0 +1,22 @@
+import { useState } from "react";
+
+
+const useToggle =(defaultValue:boolean)=>{
+
+
+     const [toggledElement, setToggledElement] = useState<boolean>(defaultValue);  
```

Comment:
I suggest renaming them to `[value,setValue]`

Reply:
done